packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue

```diff
@@ -79,6 +86,7 @@ export default {
 </script>
 
 <style lang="scss" module>
+@import '../assets/scss/navigation.scss';
```

Comment:
Out of interest – what in this SCSS file is used in this component?

Can see `$countrySelector-flag-width` and `$countrySelector-flag-height` – is that the only parts? (just wondering as could move those out into `common.scss` if that's the case rather than importing all the other styles too).